build-system/tasks/bundle-size/index.js

```diff
@@ -88,14 +89,13 @@ async function checkResponse(response, ...successMessages) {
 }
 
 /**
- * Helper that lazily imports node-fetch, and does a JSON POST request.
```

Comment:
Can we keep it lazy? It was to reduce Node startup time

Reply:
Not really necessary unless flame graphs (`npx 0x amp.js`) show that this is as slow to load as the previous library.